pkg/monitoring/rules/recordingrules/operator.go

```diff
@@ -0,0 +1,55 @@
+package recordingrules
+
+import (
+	"github.com/machadovilaca/operator-observability/pkg/operatormetrics"
+	"github.com/machadovilaca/operator-observability/pkg/operatorrules"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+const (
+	CommonTemplatesRestoredIncreaseQuery   = "sum(increase(kubevirt_ssp_common_templates_restored_total{pod=~'ssp-operator.*'}[1h]))"
+	TemplateValidatorRejectedIncreaseQuery = "sum(increase(kubevirt_ssp_template_validator_rejected_total{pod=~'virt-template-validator.*'}[1h]))"
+)
+
+var operatorRecordingRules = []operatorrules.RecordingRule{
```

Comment:
Can you change this variable into a function call? Similar to `alerts/operator.go`. Then it would not be possible to modify the rules by mistake.